book/src/install.md

```diff
@@ -168,16 +170,18 @@ a c++ compiler to be installed, for example `gcc-c++`.
 
 #### Linux and macOS
 
-Either set the `HELIX_RUNTIME` environment variable to point to the runtime files and add it to your `~/.bashrc` or equivalent:
+The **runtime** directory is one below the Helix source, so either set a
+`HELIX_RUNTIME` environment variable to point to that directory and add it to
+your `~/.bashrc` or equivalent:
 
 ```sh
 HELIX_RUNTIME=/home/user-name/src/helix/runtime
 ```
 
-Or, create a symlink in `~/.config/helix` that links to the source code directory:
+Or, create a symbolic link:
 
 ```sh
-ln -s $PWD/runtime ~/.config/helix/runtime
+ln -s  /home/user-name/src/helix/runtime ~/.config/helix/runtime
```

Comment:
I think we should continue to use `~/src` for these from the new intro in the building from source section. `/home/<name>` might be confusing for macos users since home directories are under `/Users/<name>` in macos